custom/icds_reports/utils/aggregation_helpers/ccs_record_monthly.py

```diff
@@ -195,4 +196,5 @@ def aggregation_query(self):
     def indexes(self):
         return [
             'CREATE INDEX ON "{}" (awc_id, case_id)'.format(self.tablename),
+            'CREATE INDEX ON "{}" (person_case_id, add, case_id)'.format(self.tablename)
```

Comment:
Do these extra indexes actually change the query/help?

From what I can tell the query will have the entire (or almost entire) ccs_record and child_health monthly tables in memory (or at least in the query space), so it will probably use a hash join instead of the index. Is that correct?